.github/workflows/build_mpv_binary_and_deb_package.yml

```diff
@@ -0,0 +1,139 @@
+name: Build mpv binary and create debian package
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y \
+            build-essential \
+            clang \
+            yasm \
+            git \
+            meson \
+            ninja-build \
+            autoconf \
+            automake \
+            libtool \
+            pkg-config \
+            libfreetype6-dev \
+            libfribidi-dev \
+            libfontconfig1-dev \
+            libasound2-dev \
+            pulseaudio \
+            libx11-dev \
+            libxss-dev \
+            libvdpau-dev \
+            libgl-dev \
+            libxv-dev \
+            libjpeg-dev \
+            libssl-dev \
+            libx264-dev \
+            libmp3lame-dev \
+            libfdk-aac-dev \
```

Comment:
encoders are not needed either

Reply:
the encoders `libmp3lame-dev, libx264-dev` are necessary for creating `.deb` package, that's why I added. If I don't I get the following error. But I will create a separate workflow for building the debian package as you have suggested.

```
dpkg-buildpackage: info: host architecture amd64
dpkg-source: info: using options from mpv-build/debian/source/options: --compression=xz --compression-level=best
dpkg-checkbuilddeps: error: Unmet build dependencies: libmp3lame-dev libx264-dev
dpkg-buildpackage: warning: build dependencies/conflicts unsatisfied; aborting
dpkg-buildpackage: warning: (Use -d flag to override.)
```